### PR TITLE
updating to latest dacfx for some bug fixes

### PR DESCRIPTION
**Pull request by udeeshagautam**
updating dacfx for a few upstream bug-fixes in ADS projects

### CODE REVIEW

**Review by kisantia — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
@udeeshagautam I'll leave it up to you what testing exactly you do for DacFX since it's your component - but it would be appreciated if you filled out the form the bot replied with for future updates to at least show that something was done. We want to ensure that we're consistent with the process for all our external components (especially important ones like DacFX)

**Comment by Charles-Gagnon** [comment]
Also I just noticed that the Integration tests have been failing.

https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=95312&view=results

Now the issue isn't from anything in your changes - but in general please don't merge in unless all checks are green. And feel free to reach out if you have any questions on failures. Thanks!

**Pull request merged**
